tests/e2e/helper.ts

```diff
@@ -0,0 +1,22 @@
+import { Aptos, AptosConfig, Network, NetworkToFaucetAPI, NetworkToIndexerAPI, NetworkToNodeAPI } from "../../src";
+
+/**
+ * This function attempts to read env vars first to configure an Aptos client. If they
+ * aren't set, it falls back to the values for Network.LOCAL. Use this whenever you
+ * want a client to use with a local testnet.
+ */
+export function getAptosClient(additionalConfig?: Partial<AptosConfig>): { aptos: Aptos; config: AptosConfig } {
+  // TODO: Do I need to use the dotenv lib for this?
+  const nodeApiUrl = process.env.APTOS_NODE_API_URL || NetworkToNodeAPI.local;
+  const indexerApiUrl = process.env.APTOS_INDEXER_API_URL || NetworkToIndexerAPI.local;
+  const faucetApiUrl = process.env.APTOS_FAUCET_API_URL || NetworkToFaucetAPI.local;
+  const config = new AptosConfig({
+    network: Network.LOCAL,
```

Comment:
l think we should only change the network. Running tests with fullnode points to server A and indexer points to server B would fail.

Also, assume I want to test a complete different network, for example devnet, I should just be able to set the network name and not each endpoint. As it is now, to test against devnet I would need to change fullnode/indexer/faucet specifically - a bit annoying.

Reply:
I think what we would want to do is
```
const network = process.env.APTOS_NETWORK || Network.LOCAL;
 const nodeApiUrl = process.env.APTOS_NODE_API_URL || NetworkToNodeAPI.local;
  const indexerApiUrl = process.env.APTOS_INDEXER_API_URL || NetworkToIndexerAPI.local;
  const faucetApiUrl = process.env.APTOS_FAUCET_API_URL || NetworkToFaucetAPI.local;
```

and then for you to use it in the other PR, you could set the network to `Network.CUSTOM` along with all other endpoints

Reply:
The flexibility is intentional, what if I want to test against a different node API but use the normal indexer API? Or in my case, they're using a different port than Network.LOCAL says.

We could add the env var that we have in the examples as well, `APTOS_NETWORK`. I'll push that up so you can see what I mean.